src/common/BaseAnnotator.scss

```diff
@@ -2,14 +2,14 @@
     font-weight: normal;
     font-family: 'Lato';
     font-style: normal;
-    src: local('Lato Regular'), local('Lato-Regular'), url('https://cdn01.boxcdn.net/fonts/1.0.2/lato/Lato-Regular.woff2') format('woff2'), url('https://cdn01.boxcdn.net/fonts/1.0.2/lato/Lato-Regular.woff') format('woff');
+    src: local('Lato Regular'), local('Lato-Regular'), url('https://cdn01.boxcdn.net/fonts/1.0.17/lato/Lato-Regular.woff2') format('woff2'), url('https://cdn01.boxcdn.net/fonts/1.0.17/lato/Lato-Regular.woff') format('woff');
```

Comment:
Could we import `fonts.scss` from BUIE, instead? It'd be one fewer place to keep track of the version.